atomic_reactor/plugins/build_source_container.py

```diff
@@ -88,7 +87,8 @@ def run(self):
         if maven_source_exists and not os.listdir(maven_source_data_dir):
             self.log.warning("Maven source directory '%s' is empty", maven_source_data_dir)
 
-        image_output_dir = tempfile.mkdtemp()
+        os.makedirs(os.path.join(self.workflow.source.workdir, 'output'), exist_ok=True)
+        image_output_dir = os.path.join(self.workflow.source.workdir, 'output')
```

Comment:
Could the order of these 2 be replaced? so we would have:
```
image_output_dir = os.path.join(self.workflow.source.workdir, 'output')
os.makedirs(image_output_dir, exist_ok=True)
```

Reply:
updated.